doc/src/manual/methods.md

```diff
@@ -241,23 +240,18 @@ julia> g(2, 3.0)
 8.0
 
 julia> g(2.0, 3.0)
-ERROR: MethodError: g(::Float64, ::Float64) is ambiguous. Candidates:
-  g(x, y::Float64) in Main at none:1
-  g(x::Float64, y) in Main at none:1
```

Comment:
Yes. But it cannot be done for this one, it will show closest candidates which are defined in the sandbox module which the test is run in. So either we don't doctest this or we leave out the closes candidates.